examples/complete/variables.tf

```diff
@@ -33,3 +33,9 @@ variable "nat_gateway_enabled" {
   type        = bool
   description = "Whether to enable NAT Gateways. If false, then the application uses NAT Instances, which are much cheaper."
 }
+
+variable "ipv6_enabled" {
```

Comment:
Is this variable used in `complete/main.tf` and you just forgot to include the variable?

Reply:
I'm not sure, it looks like it was [introduced](https://github.com/masterpointio/terraform-aws-ssm-agent/commit/dfc6f9450d3c0219a11e02623a88bd443537c43b) back in June. I added the variable definition here because it is currently failing the `terraform_validate` pre-commit check.